lib/src/utils/lerp.dart

```diff
@@ -91,3 +92,23 @@ List<ScatterSpot> lerpScatterSpotList(List<ScatterSpot> a, List<ScatterSpot> b,
 List<BarChartRodStackItem> lerpBarChartRodStackList(
         List<BarChartRodStackItem> a, List<BarChartRodStackItem> b, double t) =>
     _lerpList(a, b, t, lerp: BarChartRodStackItem.lerp);
+
+List<RadarEntry> lerpRadarEntryList(List<RadarEntry> a, List<RadarEntry> b, double t) {
```

Comment:
You can use our pre-written generic function to handle it (check top lerp functions)
Like this:
```
List<RadarEntry> lerpRadarEntryList(List<RadarEntry> a, List<RadarEntry> b, double t) => 
  _lerpList(a, b, t, lerp: RadarEntry.lerp);

List<RadarDataSet> lerpRadarDataSetList(List<RadarDataSet> a, List<RadarDataSet> b, double t) =>
    _lerpList(a, b, t, lerp: RadarDataSet.lerp);
```

Reply:
It helps to have a single function to debug, and maintain.